openfecwebapp/templates/search-results.html

```diff
@@ -13,7 +13,7 @@
     <header>
       <h1 class="heading--main">Candidate and committee profiles</h1>
     </header>
-    <form action="/" method="GET" class="slab slab--inline slab--neutral">
+    <form action="" method="GET" class="slab slab--inline slab--neutral">
```

Comment:
Nope. That was causing the form to submit to the home page and not the main data landing page.